lib/signs_ui/config/utilities.ex

```diff
@@ -0,0 +1,28 @@
+defmodule SignsUi.Config.Utilities do
+  @moduledoc """
+  Utility functions for interacting with the arinc_to_realtime.json file
+  """
+
+  def clean_configs(sign_configs) do
+    registered_ids = get_realtime_ids()
+
+    :maps.filter(
+      fn key, _ ->
+        key in registered_ids
+      end,
+      sign_configs
+    )
+  end
+
+  def get_realtime_ids do
+    get_arinc_to_realtime_mapping() |> Enum.map(fn {_k, v} -> v end)
```

Comment:
If `get_arinc_to_realtime_mapping()` returns a map, you could use `Map.values/1` instead of the current `Enum.map`